features/steps/tools.py

```diff
@@ -173,15 +173,35 @@ def check_volumes_created(client, test_name):
             assert volume.status == "available", f"Volume {volume.name} not available"
             return volume.status
 
-
-def collect_ips(client):
-    print("collecting ips")
-    ports = client.network.ports()
+def collect_ips(client,logger:Logger):
     ips = []
-    for port in ports:
-        for fixed_ip in port.fixed_ips:
-            ips.append(fixed_ip["ip_address"])
-    return ips
+    assertline=None
+    floating_ips = client.network.ips()
+    for ip in floating_ips:
+        ips.append(ip.floating_ip_address)
+        logger.log_info(f"found {ip.floating_ip_address}")
```

Comment:
fix pep

Reply:
i don't really see whats wrong there? the identation is matching the rest